src/mockgyver.erl

```diff
@@ -414,10 +414,10 @@
 %%%===================================================================
 
 %% @private
-exec(MockMFAs, WatchMFAs, Fun) ->
+exec(Context, Fun) ->
```

Comment:
1. Thinking it might be good to -spec this, now that the name of the variables don't hint at what it is anymore, and/or
2. Is Context the best name here?

I'm thinking it's not the context in which we're doing this. We're mocking and watching MFAs in some context.  Something with MFA?

Reply:
I agree it would be good to spec this. I was first thinking specing should perhaps be a separate pr since there are (almost) no specs and several @specs which render doc warnings now with Erlang 24. But then I realized there is actually one -spec, so ok. Will do. Due to the name-depletion (below), the also-added type will for now be context().

Naming: Well... I seem to have run out of names at the moment. Can't find any around here. Will return to this when the stores are open again. I'll (force) push the branch with the rest of the changes for now.

Reply:
Oh, apologies for accidentally pinging! It was unintentional.

Reply:
:-)

I've pushed a side-branch called specify with type specs added to exported functions and prominent data types.

Reply:
It is now session params instead of context

Reply:
Guess we've settled on session params instead of context, so I'll try to mark this item as resolved. (Guess it is still possible to re-open if there's anything more)